pypeit/edgetrace.py

```diff
@@ -696,6 +702,9 @@ def auto_trace(self, bpm=None, debug=False, show_stages=False):
               traces* are based on the result of :func:`peak_refine`.
             - Synchronize the left and right traces into pairs that
               define slit apertures using :func:`sync`.
+            - Use :func:`maskdesign_matching` to add match the slit
```

Comment:
Should this function maskdesign_matching be a method in spectrograph? It seems to me it should.